code/modules/research/rdconsole.dm

```diff
@@ -205,7 +205,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 	usr.set_machine(src)
 	if(href_list["menu"]) //Switches menu screens. Converts a sent text string into a number. Saves a LOT of code.
 		var/temp_screen = text2num(href_list["menu"])
-		if(temp_screen <= 1.1 || (3 <= temp_screen && 4.9 >= temp_screen) || src.allowed(usr) || emagged) //Unless you are making something, you need access.
+		if(temp_screen <= 1.1 || (2 <= temp_screen && 4.9 >= temp_screen) || src.allowed(usr) || emagged) //Unless you are making something, you need access.

```

Comment:
To be fully honest it's annoying and confusing as fuck yeah, but that's work for another day